website/oauth/models/__init__.py

```diff
@@ -489,7 +491,7 @@ class ApiOAuth2Application(StoredObject):
     name = fields.StringField(index=True, required=True, validate=[string_required, MaxLengthValidator(200)])
     description = fields.StringField(required=False, validate=MaxLengthValidator(1000))
 
-    date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow,
+    date_created = fields.DateTimeField(auto_now_add=timezone.now,
```

Comment:
`date_created` shouldn't ever be set other than on creation, unless there's a migration to a new kind of model and we want to keep the old data (e.g. the `PreprintService` migration, which set it based on the `node.preprint_created` field). This was accomplished with pymongo's `database.find_and_modify(<selector>, {'date_created': node.preprint_created})`.

Is there a similar "this shouldn't ever happen but just in case"-type workaround for manipulating psql data?